src/averageAges.js

```diff
@@ -34,7 +46,24 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  let filteredPeople;
```

Comment:
Не самое лучшее решение, обьявлять переменную без значения. Когда нужно положить в переменную значение в зависимости от какого-то условия, для такой ситуации отлично подходит тернарник
```suggestion
  let filteredPeople = withChildren
    ? ...*your code*
    : ...*your code*
```